### DISCUSSION TITLE
Iris Contributors

### DISCUSSION THREAD

**Discussion by bjlittle** [SciTools/iris#4341] [Ideas]
I recently noticed that [pyproj](https://github.com/pyproj4/pyproj) has a dedicated section in their `README.md` listing their [Contributors](https://github.com/pyproj4/pyproj#contributors-).

They've used [All Contributors](https://allcontributors.org/) to assist with the management and automation of the process, which has an associated [bot](https://allcontributors.org/docs/en/bot/overview) that can be configured for repo maintainers to easily include and update the status of contributors.

Is this a gesture that we'd want to adopt and share with the Iris community?

I certainly wouldn't suggest that we populate our `README.md`, but rather a specific page of the documentation perhaps?

**Comment by tkknight** [comment]
Looks like a nice potential addition to the docs, especially as it uses existing tooling to generate the list of contributors.

**Comment by wjbenfold** [comment]
A related idea that I've had is that we could give some recognition to people who help other people out on Yammer - maybe at the end of the year an AVD general shout-out to everyone who's contributed support there? Would have the added benefit of publicising it as a source of support
